legal/legal/NOTICE.md

```diff
@@ -15,6 +15,7 @@ of the Eclipse Public License 1.0 which is available at https://www.eclipse.org/
 
 * Copyright 2016-2018 Bosch Software Innovations GmbH
 * Copyright 2016-2018 Red Hat, Inc.
+* [Alfusainey Jallow] (https://github.com/Alfusainey/)
```

Comment:
please use the same format as the entries above, i.e. add the copyright notice.  you may still add the link to your GitHub profile, though.

Reply:
thanks. then how about `Copyright 2018 [Alfusainey Jallow] (https://github.com/Alfusainey/)`? or should it include an end date?